docs/exts/provider_yaml_utils.py

```diff
@@ -69,6 +69,8 @@ def load_package_data() -> list[dict[str, Any]]:
             jsonschema.validate(provider, schema=schema)
         except jsonschema.ValidationError:
             raise Exception(f"Unable to parse: {provider_yaml_path}.")
+        if provider["suspended"]:
```

Comment:
Should we update the docstring to reflect that the list contains only non-suspended providers?

Reply:
Yeah. I thougt about it - we will have to update it in a few places  (for example in the documentation of the provider itself) - but that is something for a separate PR. I truly hope the effect of having suspension process is that maintainers of suspended "properties" will be russhing to fix the problem and resume the providers, I want to see the effect of just letting them know without affecting the documentation (yet). 

If it won't work this time, yeah the docs generation in a few places should get some notices that suspended providers are not included.